alchemist-loading/src/test/resources/testCustomExport.yml

```diff
@@ -1,5 +1,7 @@
-incarnation: sapere
-
+incarnation: protelis
 export:
-   - type: MeanSquaredError
-     parameters: ["A", "A", product, "A", "A"]
+  - type: CSVExporter
+    parameters: ["tesing_custom_export"]
```

Comment:
We should send this to a temporary folder. At this point, we might want to have a sensible default to a temp folder...

Reply:
Done.